### PR TITLE
[bitnami/memcadhed] Memcached persistence is redundant

### PR DESCRIPTION
**Pull request by kfirfer**
Memcached persistence is redundant due to its designed place all data on RAM

### CODE REVIEW

File: bitnami/memcached/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Infrastructure
 apiVersion: v2
-appVersion: 1.6.9
+appVersion: 1.7.0
```
Comment (javsalgar) [inline bitnami/memcached/Chart.yaml:4]: You need to bump the `version` not the `appVersion`

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thanks for the PR! Please check my comment

File: bitnami/memcached/Chart.yaml (hunk 1 above)
Comment (kfirfer) [reply bitnami/memcached/Chart.yaml:4]: ahh rights! thanks,  I will change it right away

**Comment by kfirfer** [comment]
@javsalgar changed it to version instead appVersion

**Review by javsalgar — APPROVED** [review]

**Pull request merged**